unlock-app/src/components/interface/Error.js

```diff
@@ -6,52 +6,45 @@ import { resetError } from '../../actions/error'
 import Buttons from './buttons/layout'
 import ErrorMessage from '../helpers/ErrorMessage'
 
-export const Error = ({ error, close }) => {
-  if (!error) {
+export const Errors = ({ errors, close }) => {
+  const content = errors.map(error => (
+    <Wrapper key={error}>
+      <Error>{ErrorMessage(error)}</Error>
+      <SecondColumn cols={errors.length ? errors.length : 1}>
```

Comment:
I am confused about `SecondColumn`, could you clarify what that is for?

Reply:
this was necessary when we were glomming several errors together. It just died a painful death